.github/workflows/bats.yaml

```diff
@@ -164,7 +164,8 @@ jobs:
       shell: pwsh
       run: >-
         wsl.exe -d Debian --exec /usr/bin/env DEBIAN_FRONTEND=noninteractive
-        sh -c 'apt-get update && apt-get install --yes coreutils curl'
+        sh -c 'apt-get update && apt-get install --yes
+        bsdextrautils coreutils curl' # spellcheck-ignore-line
```

Comment:
Extra indent means the `>-` wouldn't be in effect (and this will be passed to `wsl.exe` as a multi-line string).  I will see if I make it less ugly (probably by adding the word to the spell checker) though.